mewarpx/mewarpx/emission.py

```diff
@@ -357,6 +358,12 @@ def __init__(self, emitter, species, npart_per_cellstep, T,
         self.use_Schottky = use_Schottky
         self.wfac = wfac
 
+        # get values from emitter's conductor if not specified
```

Comment:
```suggestion
        # Get values from the emitter and its conductor if not specified
```
Since `T` comes directly from the emitter, not the conductor.